docs/deep-dive/02-dd-create-git-function-serverless.md

```diff
@@ -120,64 +120,53 @@ Follows these steps:
 
     </details>
     <details>
-    <summary label="console-ui">
-    Console UI
+    <summary label="busola-ui">
+    Busola UI
     </summary>
 
+>**NOTE:** Busola is not installed by default. Follow the [instructions](https://github.com/kyma-project/busola/blob/main/README.md) to install it with npx.
+
 1. Create a Namespace or select one from the drop-down list in the top navigation panel.
 
 2. Create a Secret (optional).
 
   If you use a secured repository, you must first create a Secret with either basic (username and password or token) or SSH key authentication to this repository in the same Namespace as the Function. To do that, follow these sub-steps:
 
-  - On your machine, create this YAML file with one of these Secret definitions:
+  a. Open your Namespace view. In the left navigation panel, go to **Configuration** > **Secrets** and select the **Create Secret** button.
```

Comment:
Could you indent these points?

Reply:
Done. I hope it will be displayed better on the review now.